server/mlabns/util/lookup_query.py

```diff
@@ -85,6 +85,12 @@ def _set_user_defined_ip_and_af(self, request):
                                           default_value=None)
       self.user_defined_af = request.get(message.ADDRESS_FAMILY,
                                          default_value=None)
+      # If the user-defined IP address matches the GAE IP address (i.e. the user
```

Comment:
My understanding is that this adheres to the style guide:
https://engdoc.corp.google.com/eng/doc/devguide/py/styleguide.shtml?cl=head#s3.2-line-length
Style guide says max length is 80, this line is 80. gpylint doesn't show an error for this line.